.eslintrc.json

```diff
@@ -25,6 +25,7 @@
 		"indent": ["error", "tab"],
 		"linebreak-style": ["error", "unix"],
 		"quotes": ["error", "single"],
-		"semi": ["error", "never"]
+		"semi": ["error", "never"],
+		"react/prop-types": 0
```

Comment:
This globally disables prop-types, right? Why do we want that?

Reply:
You can use Generics. https://www.typescriptlang.org/docs/handbook/generics.html
 And in that case, the use of types on props is unnecessary. I mean this: `React.FC<Props>` 
It is up to you to discuss whether it is better to use generics or input property typing

Reply:
TypeScript checks component properties types at build time (when compaling to JS) and prop-types checks them at run-time. I think we don't need this double checking when we've setuped very scrict TS rules, that should help us to avoid those kind of errors. :grin:

https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application